backend/api/views.py

```
@@ -0,0 +1,20 @@
+from django.shortcuts import render
+
+# Create your views here.
+# api/views.py
+from rest_framework.views import APIView
+from rest_framework.response import Response
+
+class DemoAPIView(APIView):
+    def get(self, request):
```

Ruff (0.8.2)

Missing return type annotation for public function `get`

(ANN201)